code/clue.js

```diff
@@ -9,51 +9,179 @@ const mrGreen = {
+  image: 'assets/white.png',
+  occupation: 'Headmaster'
+}
+
 
-// CREATE OBJECTS FOR ALL THE WEAPONS, ADD MORE CHARACTERISTICS TO THE WEAPONS IF YOU LIKE.
+// CREATED OBJECTS FOR ALL THE WEAPONS
 
-const rope = {
-  name: 'rope',
+const Rope = {
+  name: 'Rope',
   weight: 10
 }
+const Knife = {
+  name: 'Knife',
+  weight: 1
+}
+const Candlestick = {
+  name: 'Candlestick',
+  weight: 15
+}
+const Dumbbell = {
+  name: 'Dumbbell',
+  weight: 30
+}
+const Poison = {
+  name: 'Poison',
+  weight: 3
+}
+const Axe = {
+  name: 'Axe',
+  weight: 16
+}
+const Bat = {
+  name: 'Bat',
+  weight: 9
+}
+const Trophy = {
+  name: 'Trophy',
+  weight: 50
+}
+const Pistol = {
+  name: 'Pistol',
+  weight: 8
+}
 
-// THE ROOMS ONLY HAS A NAME SO NO NEED FOR OBJECTS THERE.
-
-// NOW GROUP ALL SUSPECTS, WEAPONS AND ROOMS IN ARRAYS LIKE THIS:
-
+// SUSPECTS
 const suspects = [
-  mrGreen,
-  mrsWhite
-  // ...  and the rest
+  mrGreen, mrsWhite, professorPlum, missScarlet, mrsPeacock, colonelMustard, 
 ]
 
-const weapons = []
+// WEAPONS 
+const weapons = [
+   Rope, Knife, Candlestick, Dumbbell, Poison, Axe, Bat, Trophy, Pistol
+]
 
-const rooms = []
+// ROOMS
+const rooms = [
+  'Dining room', 'Conservatory', 'Kitchen', 'Study', 'Library', 'Billiard room', 'Lounge', 'Ballroom', 'Hall', 'Spa', 'Living room', 'Observatory', 'Theater','Guest house', 'Patio'
+]
 
 // THIS FUNCTION WILL RANDOMLY SELECT ONE ITEM FROM THE ARRAY THAT YOU PASS IN TO THE FUNCTION.
-// YOU DON'T NEED TO CHANGE THIS, JUST TRY TO UNDERSTAND IT. AND HOW TO USE IT.
 const randomSelector = array => {
   return array[Math.floor(Math.random() * array.length)]
 }
 
-// CREATE AN OBJECT THAT KEEPS THE MYSTERY.
-// With a killer, a weapon and a room.
-// The values will be set later.
+// CREATED AN OBJECT THAT KEEPS THE MYSTERY.
+
+let mystery = { 
+  killer: "",
+  weapon: "",
+  room: "",
+}
+
+
+
+// CREATE FUNCTIONS pickKiller
 
-// This function will be invoked when you click on the killer card.
 const pickKiller = () => {
-  // This will randomly select a killer from the suspects. And add that to the mystery object.
+  // This will randomly select a kiler from the suspects. And add that to the mystery object.
   mystery.killer = randomSelector(suspects)
+  // This will change the background color of the card to the one connected to the chosen killer and show the full name of the killer. 
+  document.getElementById("killerCard").style.background = mystery.killer.color 
+  document.getElementById('killerImage').src = mystery.killer.image
+  document.getElementById('killerName').innerHTML = `Name: ${mystery.killer.firstName} ${mystery.killer.lastName}`
+  document.getElementById('killerOccupation').innerHTML = mystery.killer.occupation
+  document.getElementById('killerAge').innerHTML = ` Age: ${mystery.killer.age}`
+  document.getElementById('killerDescription').innerHTML = mystery.killer.description
+}
+
+document.getElementById('killerCard').addEventListener('click', pickKiller)
+
```

Comment:
Maybe you could have added a removeEventListener on all three cards? That would have prevented the player from being able to click  a card several times. That would also have been helpful if you wanted to add a loader since removing an event listener allows you to control the flow. Making elements appear and disappear.